packages/kolibri-common/components/Cards/AccessibleResourceCard.vue

```diff
@@ -0,0 +1,131 @@
+<template>
+
+  <div>
```

Comment:
**[required]** Here let's remove this outer `<div>`. It's not needed, and more importantly `AccesibleResourceCard/KCard` as `<li>` will need to be a direct child of `KCardGrid` as `<ul>`